wagtailmenus/tests/test_models.py

```diff
@@ -276,3 +279,22 @@ def test_linkpage_redirects_when_served(self):
             response,
             '/superheroes/marvel-comics/spiderman/?somevar=value'
         )
+
+
+class TestNoAbsoluteUrlsPage(TestCase):
+    fixtures = ['updated_test.json']
```

Comment:
I added a new model to trigger the `DeprecationWarning`s. Wasn't sure how you want to handle updating fixtures, so I generated a new fixture. Tried to limit it to the apps/models that the are in `test.json`. Generated it using 
```
python manage.py dumpdata tests wagtailmenus sites wagtailcore.site wagtailcore.page -a --indent 4 --natural-foreign --natural-primary
```